.eslintrc.json

```diff
@@ -41,7 +41,7 @@
         "@typescript-eslint/no-non-null-assertion": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
 
-        "simple-import-sort/sort": "error",
+        "simple-import-sort/imports": "error",
```

Comment:
`"simple-import-sort/sort"` was replaced in favor of:
` "simple-import-sort/imports"` and  `"simple-import-sort/exports"`

More info https://github.com/lydell/eslint-plugin-simple-import-sort/blob/v7.0.0/CHANGELOG.md